client/src/utils/timeFormat.js

```diff
@@ -1,3 +1,8 @@
+/**
+ * Create a string representation of date using internal standard
+ * @param {Date} timeDateValue - A date object
+ * @returns {string}
+ */
 export function formatDate(timeDateValue) {
```

Comment:
Actually, a Date class instance is passed… not a simple "value".

Semantics. I'll rename these parameters in this file if that sounds good to y'all.

Reply:
Okie. Updated in a32c47d:

- `timeDateValue` → `dateTime`
- `dateTime` → `DateTimeValue`
    - I had forgotten to capitalize this [`@typedef`](https://jsdoc.app/tags-typedef.html), and I have now added the word `Value` for distinction from `Date`.

Reply:
Ticket Created: [FP-996: Update Date Instantiations with Util Function Calls](https://jira.tacc.utexas.edu/browse/FP-993)

Sorry. Normally I'd tackle this. But I am trying to keep myself off of Portal, so I can focus on CMS.